Configurations/10-main.conf

```diff
@@ -602,8 +610,9 @@ my %targets = (
         inherit_from     => [ "BASE_unix" ],
         CC               => "gcc",
         CFLAGS           => "-O3",
-        cppflags         => "-D_ENDIAN -DBN_DIV2W -D_POSIX_SOURCE -D_SOCKET_SOURCE",
-        includes         => add("/SYSLOG/PUB"),
+        cppflags         => "-D_POSIX_SOURCE -D_SOCKET_SOURCE",
+        includes         => [ "/SYSLOG/PUB" ],
+        lib_cppflags     => "-D_ENDIAN -DBN_DIV2W",
```

Comment:
BTW, @dot-asm, have you noticed this one?  I wonder what kind of endianness that is!
This has been with us a long time, it's there in 1.0.2's Configure as well...

Reply:
This ought to be just a typo. Note that not defining either -DL_ENDIAN or -DB_ENDIAN does work, so -D_ENDIAN simply means nothing.  Just omit it. [I think I recall noting it at some earlier point, but I suppose I've suppressed it.]

Reply:
Done as separate PRs (mostly to get it in all active branches)

Reply:
I've approved one for master, but now it occurs to me that it will cause conflict. Since it had to be separate request per branch, maybe you can simply omit #5564 and fix it up here.

Reply:
Nah, I did the rebase and fix conflict dance...